domains/multi-net.glitched-da-kitty-cat.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "GlitchedDaKittyCatSchoolaccount",
+        "email": "glitcheddacat@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://privacy-matters.vercel.app/"
```

CNAME should not contain protocol or `/`
site does not seem developer related
```suggestion
        "CNAME": "privacy-matters.vercel.app"
```